test/integration/generalLedger.js

```diff
@@ -0,0 +1,34 @@
+
+/* global expect, chai, agent */
+/* jshint expr : true */
+
+const helpers = require('./helpers');
+
+/*
+ * The /journal API endpoint
+ */
+describe('(/general_ledger) API endpoint', function () {
+
+  /**
+   * object containing different format of transaction to send to the
+   *server in order to get transactions or records
+   **/
+  var parameters = {
+    transactionToPost : {params : {transactions : ['TRANS2']}},
+    allRecords : {number : 2},
```

Comment:
For clarity, it might be better to separate out the parameters you will send, and those you expect to receive.  Something like:
```js
const parameters = {
  transactions : [ 'TRANS2']
};

// the number of rows we expect to find in the GL
const NUM_ROWS_GENERAL_LEDGER = 2;
```
feels more natural to me.